build.gradle.kts

```diff
@@ -198,11 +197,6 @@ tasks {
 			runtimeClasspath += sourceSets.main.get().output
 		}
 	}
-	project(":demo") {
-		this.sonar {
-			isSkipProject = true
-		}
-	}
```

Comment:
Why did you delete this part?

Reply:
See https://github.com/junit-pioneer/junit-pioneer/pull/805/commits/299c95ef51c0737c9bebd757981973d02e7586a4

Reply:
Is the `:demo` project used in any way? I couldn't find any scripts or references to it in the repo other than definition and exclusion from sonar.

Reply:
@Bukama please answer https://github.com/junit-pioneer/junit-pioneer/pull/805#issuecomment-1915280798, and/or resolve this. From the code alone I don't see `:demo` usages.